test/integration-tests/IntegrationTests.Helpers/TestHelper.cs

```diff
@@ -3,12 +3,16 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using IntegrationTests.Helpers.Models;
 using Xunit.Abstractions;
 
 namespace IntegrationTests.Helpers
 {
     public abstract class TestHelper
     {
+        // Warning: Long timeouts can cause integer overflow!
+        private static readonly TimeSpan DefaultProcessTimeout = TimeSpan.FromMinutes(60);
```

Comment:
Do we really want to have that long timeout for the sample application?
Maybe 5 minutes?